### PR TITLE
Adding compatibility with Python 3.9

### PR DESCRIPTION
**Pull request by AlbertoBittes**
# Description

Few changes made to ensure compatibility with Python 3.9. 

Fixes # (issue)

Additional dependencies introduced.
- None, added compatibility with Python 3.9

## Type of change

Please select what type of change this is.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature 
    - [x] Non-breaking change which adds functionality
    - [ ] Breaking change fix or feature that would cause existing functionality to not work as expected

## Other changes

- [x] This change requires a documentation update

Add V3.9 here:
image

# Testing

All F4Enix tests passed with Python 3.9, 3.10, 3.11

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] General testing 
    - [x] New and existing unit tests pass locally with my changes
    - [ ] Coverage is >80%

### CODE REVIEW

File: f4enix/input/ww_gvr/ww_parser.py
Comment (dodu94) [inline f4enix/input/ww_gvr/ww_parser.py:1]: Are we sure here we can work without the default arguments? ( = 1)

**Review by dodu94 — CHANGES_REQUESTED** [review]
Please check the only comment I had

**Comment by dodu94** [comment]
Also please merge developing into this pull request first as there are conflicts before the merging

File: f4enix/input/ww_gvr/ww_parser.py
Comment (AlbertoBittes) [reply f4enix/input/ww_gvr/ww_parser.py:1]: I discussed it with Alvaro, who made it, it is OK

**Pull request merged**